pkg/utils/full_sync_worker.go

```diff
@@ -31,7 +31,7 @@ func NewFullSyncThread(interval time.Duration) *FullSyncThread {
 	return &FullSyncThread{
 		Stopped:       false,
 		ShutdownChan:  make(chan string),
-		QuickSyncChan: make(chan string),
+		QuickSyncChan: make(chan struct{}, 2),
```

Comment:
Why do we need a buffered channel here?

Reply:
It was needed earlier as it was a blocking call, but I think we can change it back to 1 now, thanks